docs/kyma/04-04-cluster-installation.md

```diff
@@ -45,14 +45,28 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16.3 or higher
 
   </details>
+  <details>
+  <summary label="IBM Cloud">
+  IBM Cloud
+  </summary>
+
+  - [IBM Cloud](https://cloud.ibm.com/login) account
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+  - [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started)
+
+  >**NOTE:** Running Kyma on IBM Cloud requires three [`b3c.4x16` machines](https://cloud.ibm.com/docs/containers?topic=containers-planning_worker_nodes). Create these machines when you complete the **Prepare the cluster** step.
+
+  >**CAUTION:** To install Kyma on IBM Cloud successfully, you must lighten it a bit. [Disable](#configuration-custom-component-installation-remove-a-component) the Logging component from the default installation before you proceed to the **Install Kyma** step.
```

Comment:
I would rather say this is experimental and we know that for now Logging component doesn't work, but we are not sure what about others. So Logging should be an example.